src/SecTester.Core/Logger/ColoredConsoleFormatter.cs

```diff
@@ -0,0 +1,46 @@
+using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.IO;
+using System.Linq;
+using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Logging.Abstractions;
+using Microsoft.Extensions.Logging.Console;
+using Microsoft.Extensions.Options;
+using SecTester.Core.Utils;
+
+namespace SecTester.Core.Logger;
+
+public class ColoredConsoleFormatter : DefaultConsoleFormatter
+{
+  const string DefaultForegroundColor = "\x1B[39m\x1B[22m";
+
+
+  public ColoredConsoleFormatter(IOptionsMonitor<DefaultConsoleFormatterOptions> options, SystemTimeProvider systemTimeProvider)
+    : base(nameof(ColoredConsoleFormatter), options, systemTimeProvider)
+  {
+  }
+
+  protected override void WriteHeader<TState>(
+    in LogEntry<TState> logEntry,
+    IExternalScopeProvider? scopeProvider,
+    TextWriter textWriter)
+  {
+      textWriter.Write(GetForegroundColorAnsiCode(logEntry.LogLevel));
+      textWriter.Write(FormatHeader(logEntry.LogLevel));
+      textWriter.Write(DefaultForegroundColor);  
+  }
+
+  static string GetForegroundColorAnsiCode(LogLevel level) =>
+    level switch
+    {
+      LogLevel.Critical  => "\x1B[1m\x1B[31m",  //ConsoleColor.Red

```

Comment:
The same comment relates to each statement below
```suggestion
      LogLevel.Critical  => "\x1B[1m\x1B[31m",  // ConsoleColor.Red

```